rotkehlchen/chain/evm/decoding/gearbox/gearbox_cache.py

```diff
@@ -157,11 +160,26 @@ def register_token(evm_inquirer: 'EvmNodeInquirer', token_address: str, pool: Ge
 def ensure_gearbox_tokens_existence(
         evm_inquirer: 'EvmNodeInquirer',
         all_pools: list[GearboxPoolData],
+        msg_aggregator: 'MessagesAggregator',
 ) -> list[GearboxPoolData]:
     """This function receives data about gearbox pools and ensures that lp tokens and pool coins
     exist in rotki's database."""
     verified_pools = []
-    for pool in all_pools:
+    all_pools_length = len(all_pools)
+    last_notified_ts = 0
+    for idx, pool in enumerate(all_pools):
+        if (current_time := ts_now()) - last_notified_ts > 5:
+            msg_aggregator.add_message(
+                message_type=WSMessageType.PROTOCOL_CACHE_UPDATES,
+                data={
+                    'protocol': CPT_GEARBOX,
+                    'chain': evm_inquirer.chain_id.to_name(),
+                    'processed': idx + 1,
```

Comment:
isn't idx here correct? WHen you start you send the message because is the frist time and you have processed 0 yet